engines/griffon/cutscenes.cpp

```diff
@@ -133,6 +139,27 @@ const char *story2[27] = {
 	"and I am free to die as I please."
 };
 
+#ifdef USE_TTS
+int textToSpeech(int nextparagraph, const char *story[], int arraysize) { 
+Common::TextToSpeechManager *_ttsMan = g_system->getTextToSpeechManager();
+if (_ttsMan != nullptr && ConfMan.getBool("tts_enabled")) {
+	if ((_ttsMan->isSpeaking()) == false) {
```

Comment:
You probably don't want this test here because it means that if it has not finished speaking the previous paragraph it will skip this paragraph, and all the remaining ones afterward (as you test for `i == nextparagraph` to call this function).

What you want to do instead is to always execute this code and queue the speech so that it speaks this paragraph as soon as it can.

If you look at https://github.com/scummvm/scummvm/blob/master/common/text-to-speech.h you can see the following (I simplified a bit to keep only the relevant code):
```c++
class TextToSpeechManager {
public:
	enum Action {
		INTERRUPT,
		INTERRUPT_NO_REPEAT,
		QUEUE,
		QUEUE_NO_REPEAT,
		DROP
	};

	bool say(const String &str, Action action, CodePage charset = kUtf8);

	bool say(const String &str, CodePage charset = kUtf8) {
		return say(str, INTERRUPT_NO_REPEAT, charset);
	}
};
```

This means you can use one of two functions to speak some text, one where you pass only the text (the one you are currently using) and that interrupts the current speech, and another one where in addition to the text you also specify an action to indicate what you want it to do if it is currently speaking.  When using that one with the `QUEUE` action you indicate that it should continue the current speech and once it has finished immediately start speaking the new text. See the documentation at [on lines 169-178](https://github.com/scummvm/scummvm/blob/master/common/text-to-speech.h#L169-L178) for more details on the actions.